TCAT/Model/BusLocation.swift

```diff
@@ -25,7 +37,7 @@ class BusLocation: NSObject, NSCoding {
     var routeID: String = ""
     var runID: Int = 0
     var speed: Int = 0
-    var tripID: Int = 0
+    var tripID: String = ""
```

Comment:
will austin return all these instance vars? if so i'd consider not having a default values for the instance vars and create an initializer that takes in all these class vars so we don't have a case of accidentally using the nonsensical default values. Or make the instance vars optional if we're not guaranteed to get all this data from the backend.

Reply:
Nearly all of them will be returned. Can sep. the ones that won't be

Reply:
i think it's better to have a crazy long init function b/c it indicates a flow. Plus it guarantees that at initialization these variables are going to be filled out and no one's going to accidentally to forgot to fill them out b/c they're in the initializer